src/account-request/dto/account-request.dto.ts

```diff
@@ -8,22 +9,37 @@ export class AccountRequestProposalDTO {
     })
     customer: string
 
-    @ApiProperty({ enum: Status })
+    @ApiProperty({ description: 'Status of the request', enum: Status, example: Status.ACCEPTED })
     status: string
 
-    @ApiProperty()
+    @ApiProperty({ 
+        description: 'Date in which the request was recorded', 
+        type: 'string', format: 'YYYY-MM-DD', example: '2020-07-01'
+    })
     date: string
 
-    @ApiPropertyOptional()
+    @ApiPropertyOptional({ 
+        description: 'How many positive opininos are required to approve the request', 
+        type: 'number', minimum: 2, maximum: 1000, example: 5, default: 3
+    })
```

Comment:
There is a typo here: should be 'opinions' instead of 'opininos'.
I like the detail in the property description.